demo/src/main/java/com/teamdev/jxbrowser/demo/Application.java

```diff
@@ -21,29 +21,30 @@
 package com.teamdev.jxbrowser.demo;
 
 import com.teamdev.jxbrowser.chromium.internal.Environment;
-import com.teamdev.jxbrowser.demo.resources.Resources;
 
 import javax.swing.*;
 import java.awt.*;
 import java.awt.event.WindowAdapter;
 import java.awt.event.WindowEvent;
 
-public class JxBrowserDemo {
+import static com.teamdev.jxbrowser.demo.resources.Resources.loadIcon;
 
-    private static void initEnvironment() throws Exception {
-        System.setProperty("apple.laf.useScreenMenuBar", "true");
-        System.setProperty("com.apple.mrj.application.apple.menu.about.name", "JxBrowser Demo");
-        UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
+final public class Application {
```

Comment:
Should be `public final class...` according to https://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-8.1.1